helpers/maker-delegate-frontend/README.md

```diff
@@ -0,0 +1,6 @@
+# Wrapper
```

Comment:
remove the word wrapper? and also don't have maker-delegate-frontend.

Reply:
maker-delegate-to-maker-delegate  should be peer to peer as that's the main phrase used.

Reply:
oh, this was due to the potential rename that hadn't been finalized, ie trade table/ peer-frontend. or whatever else it may become.